app/scripts/controllers/transactions/index.js

```diff
@@ -169,12 +169,12 @@ export default class TransactionController extends EventEmitter {
     this.getEventFragmentById = opts.getEventFragmentById;
     this.getTokenStandardAndDetails = opts.getTokenStandardAndDetails;
     this.securityProviderRequest = opts.securityProviderRequest;
-    this.getSelectedAddress = opts.getSelectedAddress;
```

Comment:
This line intrdouced recently but not needed (https://github.com/MetaMask/metamask-extension/pull/21244)

Because `getSelectedAddress` is already assigned later on  `_mapMethods`, so it will work anyway.